src/neo/Cryptography/Crypto.cs

```diff
@@ -53,9 +62,19 @@ public static bool VerifySignature(ReadOnlySpan<byte> message, ReadOnlySpan<byte
             {
                 throw new ArgumentException();
             }
+
+            ECCurve ncurve;
+
+            switch (curve)
+            {
+                case ECC.ECCurve.Curve.Secp256r1: ncurve = ECCurve.NamedCurves.nistP256; break;
+                case ECC.ECCurve.Curve.Secp256k1: ncurve = ECCurve.CreateFromValue("1.3.132.0.10"); break;
```

Comment:
It looks like `secp256k1` is not supported on windows platforms. And it will throw exception when I use ECCurve.CreateFromValue("1.3.132.0.10") for verifySignature.

![image](https://user-images.githubusercontent.com/43396869/72498447-53b0b580-386a-11ea-83ba-101e67c16a6c.png)

Some other developers have encountered the same problem https://github.com/planetarium/libplanet/issues/66#issuecomment-475122997
https://github.com/dotnet/corefx/issues/29057#issuecomment-380866702

Reply:
We can use other way

Reply:
Shall we use `Bouncy.Castle`?

Reply:
I think we should use the `Bouncy.Castle` to replace the `Neo.Cryptograph.ECC`, and we can also use `secp256k1`. 

But this matter needs @erikzhang  opinion

Reply:
`secp256k1` is required in some dApps. We need to add it. I think we can use `Bouncy.Castle`. what do you think @erikzhang ?

Reply:
Bouncy.Castle implements its own `BigInteger`. I don't like this.

Reply:
Alternatives?